src/main/play-doc/general/ReleaseNotes.md

```diff
@@ -1,5 +1,29 @@
 # ConductR release notes
 
+### 1.1.3
+
+1. The Resource Provider now outputs its view of what resources it believes are available so that operators can better diagnose failure to obtain suitable resource offers.
+
+2. The service locator should always remove the first path component when replying with a redirect and only use the first path component to identify the service name in a redirect style scenario, enabling seamless usage with HTTP clients.
+
+3. Get and load bundles endpoint should only respond to `/bundles` or `/bundles/` paths, and should not response to any other subpaths underneath.
+Prior this release, calling HTTP get on `/bundles/some/random/path` will return the bundles response. This is incorrect as 404 should be returned instead
+
+4. Introduces Eslite, a basic Elasticsearch behaving bundle that doesn't require a GiB of memory to run. Eslite is ideal for localhost testing and will be used for the default logging option by the developer sandbox.
+
+5. Updates ConductR's use of the Reactive Platform to rp16s01p05.
+
+6. Fixes consolidated logging using rsyslog instead of Elasticsearch.
```

Comment:
Perhaps reword:

"Syslog connectivity was broken. This has now been repaired."